doc/decisions/validation.md

```diff
@@ -14,14 +14,14 @@ configuration.
 
 If validation of a key fails, each plugin decides on its own
 how to handle the issue and proceed in ways that might be
-different than what is expected or desired.
+different from what is expected or desired.
 
 ## Constraints
 
 ## Assumptions
 
 While plugins should always fail and return an error if validation
-fails on kdbSet, there are be can be several different requirements
+fails on kdbSet, there can be several different requirements
```

Comment:
I don't see a reason why to weaken the statement with a modal word (can), what reason did you think of?

```suggestion
fails on kdbSet, there are several different requirements
```

Reply:
The original phrase was "are be can be", so I just deleted the part "are be" to get a grammatically correct sentence, but you are right, "are" instead of "can be" is probably a better wording in this case.